distribution/sender/routes/active-mq-publisher.xml

```diff
@@ -0,0 +1,132 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+
+    <route id="active-mq-publisher" errorHandlerRef="outBoundErrorHandler">
+    
+        <!-- TODO(luis.pinto) Make the delay configurable -->
+        <from uri="scheduler:active_mq_publisher?initialDelay=15000&amp;delay=60000" />
```

Comment:
Initial delay should be may be 2minutes I think, it gives the application and other more important routes higher priority in the bootstrap process.

Reply:
Fixed. Delays moved to configurations.